src/messagingsession/DefaultMessagingSession.ts

```diff
@@ -127,6 +128,9 @@ export default class DefaultMessagingSession implements MessagingSession {
     const queryParams = new Map<string, string[]>();
     queryParams.set('userArn', [this.configuration.userArn]);
     queryParams.set('sessionId', [this.configuration.messagingSessionId]);
+    if (this.configuration.prefetchOn === PrefetchOn.Connect) {
+      queryParams.set('prefetch-on', ['connect']);
```

Comment:
Since we are using string enum and comparing with `PrefetchOn.Connect` can we use `queryParams.set('prefetch-on', [PrefetchOn.Connect]` over hard coding 'connect' separately?